conf/rest/9.13.0/snapshot.yaml

```diff
@@ -2,10 +2,6 @@ name:                     Snapshot
 query:                    api/private/cli/snapshot/policy
 object:                   snapshot
 
-schedule:
```

Comment:
Unrelated change?

Reply:
yes added in this PR based on hardik's changes in https://github.com/NetApp/harvest/pull/778